.eslintrc.json

```
@@ -9,6 +9,7 @@
   ],
   "rules": {
     "react/react-in-jsx-scope": "off",
-    "@typescript-eslint/no-var-requires": ["off"]
+    "@typescript-eslint/no-var-requires": ["off"],
+    "react/prop-types": 0
```

```suggestion
    "react/prop-types": "off"
```

0보다 off 가 더 이해하기 쉬울 것 같아요

---

위에도 배열을 빼는게 더 좋을 것 같아요.

```
"@typescript-eslint/no-var-requires": "off",
```

---

감사합니다!